datadog/dogstatsd/base.py

```diff
@@ -29,13 +29,30 @@
 DEFAULT_HOST = 'localhost'
 DEFAULT_PORT = 8125
 
-# Tag name of entity_id
-ENTITY_ID_TAG_NAME = "dd.internal.entity_id"
```

Comment:
can you keep the definition and use it in the mapping bellow to avoid breaking changes in case someone is importing this variable.